### DISCUSSION TITLE
Keycloak resource URIs wildcard support

### DISCUSSION THREAD

**Discussion by manojvignesh** [keycloak/keycloak#22184] [Archive]
I want to create a keycloak resource with uri having dynamic path & query params.

/api/{version}/app/get?isReady={isReady}

where version and isReady can be anything.

but it's not working.

`/api/v1/app/get?*` - works
`/api/{version}/app/get?*` - doesn't work

**Comment by dwalker-sabiogroup** [comment]
Raised https://github.com/keycloak/keycloak/issues/28772 as a bug
